tests/views.py

```diff
@@ -39,6 +39,8 @@ def get(self, request, *args, **kwargs):
 
 
 class APITestView(APIView):
+    test_arg = False
```

Comment:
To be honest, I was just lazy. It was late in the day. I added a test to check whether `setup_view` is called with the correct keyword arguments. Tests for `setup_view` itself are already in place.